### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by Bumpie7**

### CODE REVIEW

**Comment by Bumpie7** [comment]
That's how it looks after current changes 
image

**Review by Sashuteak — APPROVED** [review]

File: custom-my-courses-in-other-orgs.js [hunk 1]
```diff
@@ -53,15 +54,34 @@ class CustomMyCoursesInOtherOrgs extends LocalizeLitMixin(LitElement) {
 				width: -webkit-fill-available;
 			}
 
-			d2l-input-search {
+			.input-search {
```
Comment (josephsaffron) [inline custom-my-courses-in-other-orgs.js:57]: did we need to switch this from element to class selector?

File: custom-my-courses-in-other-orgs.js [hunk 2]
```diff
@@ -53,15 +54,34 @@ class CustomMyCoursesInOtherOrgs extends LocalizeLitMixin(LitElement) {
...
 				margin-bottom: 10px;
-				width: 40%;
+				max-width: 556px;
```
Comment (josephsaffron) [inline custom-my-courses-in-other-orgs.js:59]: could you add a comment `/* comment */` with a brief note about how you arrived at this number?

**Comment by Bumpie7** [comment]
image
or 
image

File: custom-my-courses-in-other-orgs.js [hunk 3]
```diff
@@ -53,15 +54,47 @@ class CustomMyCoursesInOtherOrgs extends LocalizeLitMixin(LitElement) {
...
-				width: 40%;
 			}
 
 			.d2l-main-list {
 				position: relative;
 			}
 
+			.pin-icon {
+				height: 16px;
+				margin-right: 5px;
```
Comment (josephsaffron) [inline custom-my-courses-in-other-orgs.js:67]: Does this do what you want when rtl is on?

File: custom-my-courses-in-other-orgs.js [hunk 4]
```diff
@@ -112,19 +146,8 @@ class CustomMyCoursesInOtherOrgs extends LocalizeLitMixin(LitElement) {
...
-						@d2l-input-search-searched=${this._handleSearch}
-					>
-					</d2l-input-search>
-					<d2l-my-organization-filter 
-						org-defined-id="${this.orgDefinedId}"
-						@d2l-my-organization-filter-change="${this._filterChanged}"
-					></d2l-my-organization-filter>
-					${needDisplayEmptyContainer ? this.renderEmptyCoursesContainer('searchEmptyResult') : html``}
-				</d2l-list-header>
+				${this.myCoursesInOtherOrgsList.length > 1 || this.myOrgList.length > 1 || this.textFiltered ? this.renderListHeader() : ''}
```
Comment (josephsaffron) [inline custom-my-courses-in-other-orgs.js:149]: Should '' be html``?

File: custom-my-courses-in-other-orgs.js (hunk 1 above)
Comment (Bumpie7) [reply custom-my-courses-in-other-orgs.js:57]: We want to apply our styles to classes of elements to know that styles comes from our component's css

File: custom-my-courses-in-other-orgs.js (hunk 2 above)
Comment (josephsaffron) [reply custom-my-courses-in-other-orgs.js:59]: @Bumpie7 I think you are using the wrong style of comments.  css comments are /**/

**Review by josephsaffron — APPROVED** [review]
approved, pending that we want to hide the search bar.

**Comment by Bumpie7** [comment]
image
and 
image

**Pull request merged**